### PR TITLE
Add and remove

### PR DESCRIPTION
**Pull request by youmari**
## In this branch I implemented the following :

- Add a new task by entering the name of the task and clicking at enter Icon 
- Click on a task and you will be able to edit the text of the task 
- if you remove the text from the task it will remove automatically from the list and array
- Add functionality to clear all completed tasks btn if you click it, it will remove all done tasks

### CODE REVIEW

**Review by BrianSammit — CHANGES_REQUESTED** [review]
# CHANGES REQUIRED 🔴 🛑
Hello @youmari, great job you are doing so far in this project. 💯
TO HIGHLIGHT:

- Your project is professionally 
- Your repo is professional 
- Well structured files 
- great job working on the adding, editing, and clear all actions. 

However, there are a few changes to make to move to the next project 🛠️ . Keep working hard 💪 💻 

## Required Changes ♻️

- [x] It would be nice if you add the delete a single task function. Currently, the only way you can delete a task is if you marked it as completed. 

Capture

- [x] Kindly make sure that the index of the last task added is the same as the array length. For example, if you have 5 tasks the first task must have the index = 1 and the last task index = 5. 

Capture


See my inline comment to know what changes are needed. 👇 👀 

Please make these required changes and send it again for another review 👍 💪
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
# HAPPY CODING!!! 😄 

tenor-1



**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

--------

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by youmari** [comment]
@BrianSammit  you can delete the task by remove the name of task

**Comment by youmari** [comment]
All required changes are done thank you

File: src/index.js [hunk 1]
```diff
@@ -4,31 +4,12 @@ import enter from './assets/enter.png';
 import dots from './assets/three.png';
 import recycle from './assets/recycle.png';
 import updateItemsStatus from './js/updateItems';
-import { getListItemdata } from './js/localStorageFunctions';
+import addItems from './js/addItems';
+import clearAllCompletedtasks from './clearAllCompleted';
+import editItem from './js/editAndRemoveItem';
+import trash from './assets/trash.png'
```
Comment (Dipeshtwis) [inline src/index.js:10]: - [x] Make sure you have a `trash.png` image in the assets folder. Also, make sure it is visible in right of the particular item. Currently it is not visible as well. Please fix it.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @youmari, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**

-------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (youmari) [reply src/index.js:10]: sorry I didn't pay attention it's there right now

**Comment by youmari** [comment]
when you click to edit the task the trash will show @Dipeshtwis

**Review by mikethreels — APPROVED** [review]
Hi Yassine, 

Your project is complete! it's time to merge it 🚀 :
Congratulations! 🎉 

- [x] [OPTIONAL] if you would like to make your project a little more user friendly you might want to consider changing the way the delete function works, as clicking on the task name is a little counter-intuitive, it would be a lot easier to use if the icon with the 3 dots changes to the delete icon after clicking the actual icon with 3 dots instead.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by youmari** [comment]
@mikethreels  you idea sounds cool I'll implement it  later thank you

**Pull request merged**